### PR TITLE
Removed dropping of indeces in pulp-manage-db

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

**Comment by dkliban** [comment]
ok test

File: docs/user-guide/tuning.rst [hunk 1]
```diff
@@ -176,6 +176,12 @@ dedicated hardware for larger production deployments. At this time, Pulp can be
 used with `replication <http://docs.mongodb.org/manual/replication/>`_ but does
 not support sharding.
 
+If searches for content are performing poorly, performance could be improved by adding an index
```
Comment (rbarlow) [inline docs/user-guide/tuning.rst]: s/could/may/

**Comment by rbarlow** [comment]
I believe it was agreed that we did need to drop the indices and recreate them one last time in a migration. There has been some work that drops indices in the past, and users who upgrade from an old Pulp to 2.7 will miss these index removals (thus the need for one last removal). So I think we need a migration that drops and adds one last time.

Let me know when you've done that and I can re-review.

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by beav** [comment]
This appears to address @rbarlow 's comment and looks good overall.

LGTM

**Pull request merged**